docs/resources/service_f5_vserver_endpoint.md

```diff
@@ -0,0 +1,47 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "alkira_service_f5_vserver_endpoint Resource - terraform-provider-alkira"
+subcategory: ""
+description: |-
+  
+---
+
+# alkira_service_f5_vserver_endpoint (Resource)
```

Comment:
How exactly `f5_lib` resource will work with `f5_vserver_endpoint` here?  What's the relationship between those two resources? Any more details? From the below example, `f5_vserver_endpoint` will depend on `f5_lb`.....  and those references of instances are interesting.

Reply:
so what will happen if we provision? I would guess both resources have been provisioned together otherwise, it will be timing issue? Did we try to provision?

Reply:
each endpoint is linked to an instance/s of the F5 service. 
Creation worked fine, manual provision worked fine, vserver `provision=true` had some issue, we have a fix. I'm currently testing it out. Will update here once it's done.

Reply:
👍